proto/api.proto

```diff
@@ -289,6 +289,17 @@ service Api {
     };
   }
 
+  rpc PostRouteTradeSwap(RouteTradeSwapRequest) returns (RouteTradeSwapResponse) {
+    option(google.api.http) = {
+      get: "/api/v1/amm/route-trade-swap"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      description: "routes swap(s) from supported amms";
```

Comment:
description for consistency: "Generates an unsigned transaction object for the AMM swap on the specified route"

summary for consistency: "Unsigned AMM swap transaction along route"